node/coroutines/src/ns_producer.rs

```diff
@@ -35,7 +35,7 @@ pub fn spawn_block_producer(
     client: Arc<Client>,
     consensus_rx: Receiver<ConsensusBlockProposal>,
     mempool_control_tx: Sender<MemPoolControl>,
-    receipts_tx: Sender<ReceiptBlock>,
+    _receipts_tx: Sender<ReceiptBlock>,
```

Comment:
let's remove this channels from alphanet and devnet then.

Reply:
Done.